lib/uwb/UwbDevice.cxx

```diff
@@ -198,6 +198,14 @@ UwbDevice::Reset()
     ResetImpl();
 }
 
+bool
+UwbDevice::Initialize()
+{
+    // TODO: Either make this do something useful, or make function pure virtual
+    // and have every derived class implement it
+    return true;
```

Comment:
It seems reasonable that not all derived classes will require initialization, so this makes sense as it is since this will be the default implementation if not override. You can remove the TODO.